Crate nimiq-bls doesn't build without features

The crate `nimiq-bls` doesn't build without features. This is the output building the crate without features:
```
1/4: no features
error[E0432]: unresolved import `beserial`
 --> bls/src/types/compressed_signature.rs:8:5
  |
8 | use beserial::Deserialize;
  |     ^^^^^^^^ use of undeclared crate or module `beserial`

warning: unused import: `str::FromStr`
 --> bls/src/types/compressed_public_key.rs:1:42
  |
1 | use std::{cmp::Ordering, fmt, io::Error, str::FromStr};
  |                                          ^^^^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default

warning: unused import: `ParseError`
  --> bls/src/types/compressed_public_key.rs:10:13
   |
10 | use crate::{ParseError, PublicKey};
   |             ^^^^^^^^^^

warning: unused import: `std::fmt`
 --> bls/src/types/keypair.rs:1:5
  |
1 | use std::fmt;
  |     ^^^^^^^^

warning: unused import: `std::fmt`
 --> bls/src/types/secret_key.rs:1:5
  |
1 | use std::fmt;
  |     ^^^^^^^^

error[E0599]: no function or associated item named `deserialize_from_vec` found for struct `compressed_signature::CompressedSignature` in the current scope
  --> bls/src/types/compressed_signature.rs:88:33
   |
19 | pub struct CompressedSignature {
   | ------------------------------ function or associated item `deserialize_from_vec` not found for this
...
88 |         Ok(CompressedSignature::deserialize_from_vec(&raw).unwrap())
   |                                 ^^^^^^^^^^^^^^^^^^^^ function or associated item not found in `compressed_signature::CompressedSignature`
   |
   = help: items from traits can only be used if the trait is implemented and in scope
   = note: the following traits define an item `deserialize_from_vec`, perhaps you need to implement one of them:
           candidate #1: `beserial::Deserialize`
           candidate #2: `beserial::DeserializeWithLength`

Some errors have detailed explanations: E0432, E0599.
For more information about an error, try `rustc --explain E0432`.
error: could not compile `nimiq-bls` due to 2 previous errors; 4 warnings emitted
Failed to build without features for package at /home/jsdaniel/nimiq/core-rs-albatross/bls

```

--- a/bls/src/types/compressed_public_key.rs
+++ b/bls/src/types/compressed_public_key.rs
@@ -1,4 +1,6 @@
-use std::{cmp::Ordering, fmt, io::Error, str::FromStr};
+#[cfg(feature = "beserial")]
+use std::str::FromStr;
+use std::{cmp::Ordering, fmt, io::Error};
 
 use ark_ec::AffineCurve;
 use ark_mnt6_753::G2Affine;
@@ -7,7 +9,9 @@ use ark_mnt6_753::G2Affine;
 use beserial::Deserialize;
 
 use crate::compression::BeDeserialize;
-use crate::{ParseError, PublicKey};
+#[cfg(feature = "beserial")]
+use crate::ParseError;
+use crate::PublicKey;
 
 /// The serialized compressed form of a public key.
 /// This form consists of the x-coordinate of the point (in the affine form),

--- a/bls/src/types/compressed_signature.rs
+++ b/bls/src/types/compressed_signature.rs
@@ -1,13 +1,16 @@
 use std::io::Error;
+#[cfg(feature = "beserial")]
 use std::str::FromStr;
 use std::{cmp::Ordering, fmt};
 
 use ark_ec::AffineCurve;
 use ark_mnt6_753::G1Affine;
 
+#[cfg(feature = "beserial")]
 use beserial::Deserialize;
 
 use crate::compression::BeDeserialize;
+#[cfg(feature = "beserial")]
 use crate::ParseError;
 use crate::Signature;
 
@@ -77,6 +80,7 @@ impl fmt::Display for CompressedSignature {
     }
 }
 
+#[cfg(feature = "beserial")]
 impl FromStr for CompressedSignature {
     type Err = ParseError;
 

--- a/bls/src/types/keypair.rs
+++ b/bls/src/types/keypair.rs
@@ -1,3 +1,4 @@
+#[cfg(feature = "beserial")]
 use std::fmt;
 
 #[cfg(feature = "beserial")]

--- a/bls/src/types/secret_key.rs
+++ b/bls/src/types/secret_key.rs
@@ -1,3 +1,4 @@
+#[cfg(feature = "beserial")]
 use std::fmt;
 
 use ark_ff::{UniformRand, Zero};

